lingocore/lib/widgets/common_widgets.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:flutter/material.dart';
+class ButtonWidget extends StatelessWidget {
```

Comment:
ButtonWidget ne? AnswerButton olabilir QuestionAnswerButton olabilir.
Ayrıca common_widgets.dart dosya ismi yeterince açıklayıcı değil. Neye göre common? Common ortak anlamına geliyor, kimin için ortak? Ayarlar menüsünde de mi ortak? Dosya adını düzelt.